Yoast/wordpress-seo
# Added GDPR message and new copy for signing up to the newsletter

## Summary

This PR can be summarized in the following changelog entry:

* Updates the newsletter signup page in the Configuration Wizard with some new copy and layout and adds a link to our privacy policy to better adhere to the GDPR standards.
* Fixes a bug where incorrect prop types were being passed to the Suggestions step in the Configuration Wizard.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Check out the branch and ensure you've run `composer install`, `yarn` and `grunt build`.
* Open up the Configuration Wizard and navigate to step 10.
* Determine that the copy has changed, the name field is gone and there's a message regarding the privacy policy.
* Determine that the link in the message regarding the privacy policy, properly works.

Regarding the bugfix:

* Before checking out this branch, navigate to step 11 in the Configuration Wizard.
* Check your console and notice that warnings are thrown.
* After checking out this branch and installing + building all dependencies, check step 11 again and see that the warning is gone.

## Quality assurance


Fixes #9665
